packages/ethereum-contracts/package.json

```diff
@@ -29,7 +29,7 @@
         "run-forge": "IS_FOUNDRY=true forge",
         "run-nodemon": "nodemon -e sol,js,ts -x",
         "build": "run-s build:*",
-        "build:contracts": "rm -f build/contracts/*;yarn run-truffle compile",
+        "build:contracts": "rm -f build/contracts/*;yarn run-truffle compile && yarn build:hardhat",
```

Comment:
we are making buiild time longer?!

could split them into two:


build:contracts-truffle
build:contracts-hardhat

Reply:
okay, true